src/nurse/views.py

```diff
@@ -143,6 +157,23 @@ def get_queryset(self):
         queryset = queryset.filter(patient__nurse=nurse)
         return queryset
 
+    def create(self, request, *args, **kwargs):
+        """Prevents the creation of a prescription"""
+        """if the free limit is reached without an active subscription."""
+        prescription_count = Prescription.objects.count()
+        try:
+            subscription = Subscription.objects.get(user=self.request.user)
+        except Subscription.DoesNotExist:
+            subscription = None
+        is_subscribed = subscription is not None and subscription.active
+        if not is_subscribed and prescription_count >= settings.FREE_PRESCRIPTION_LIMIT:
```

Comment:
Reuse the yet to be created `get_active_subscription()` method

Reply:
Done